### PR TITLE
Plotregion

### PR DESCRIPTION
**Pull request by shadrackkibet**

### CODE REVIEW

**Comment by shadrackkibet** [comment]
@dannyparsons  this is ready to be merged.This are just designer and layout fixes.

**Comment by maxwellfundi** [comment]
@dannyparsons  this is ready for you

**Comment by shadrackkibet** [comment]
fixes #3906

File: instat/dlgMakeDate.vb [hunk 1]
```diff
@@ -80,6 +80,7 @@ Public Class dlgMakeDate
         dctDateFormat.Add("Year(4-digit)/Month(Full Name)/Day", Chr(34) & "%Y/%B/%d" & Chr(34))
         dctDateFormat.Add("Year(4-digit)-Month(abbr)-Day", Chr(34) & "%Y-%b-%d" & Chr(34))
         dctDateFormat.Add("Year(4-digit)/Month(abbr)/Day", Chr(34) & "%Y/%b/%d" & Chr(34))
+        dctDateFormat.Add("Year(4-digit)/Doy(Julian)", Chr(34) & "%Y%j" & Chr(34))
```
Comment (dannyparsons) [inline instat/dlgMakeDate.vb]: Careful with the display text here, the format doesn't include a "/" separating year and doy so it shouldn't be there in the combo box.

Comment (shadrackkibet) [reply instat/dlgMakeDate.vb]: will remove this then.

**Comment by shadrackkibet** [comment]
@dannyparsons this is ready for you again.

File: instat/dlgMakeDate.vb [hunk 2]
```diff
@@ -80,7 +80,6 @@ Public Class dlgMakeDate
         dctDateFormat.Add("Year(4-digit)/Month(Full Name)/Day", Chr(34) & "%Y/%B/%d" & Chr(34))
         dctDateFormat.Add("Year(4-digit)-Month(abbr)-Day", Chr(34) & "%Y-%b-%d" & Chr(34))
         dctDateFormat.Add("Year(4-digit)/Month(abbr)/Day", Chr(34) & "%Y/%b/%d" & Chr(34))
-        dctDateFormat.Add("Year(4-digit)/Doy(Julian)", Chr(34) & "%Y%j" & Chr(34))
```
Comment (dannyparsons) [inline instat/dlgMakeDate.vb]: We still want this option, but the text associated with this format is wrong, it just shouldn't include the "/"

Comment (shadrackkibet) [reply instat/dlgMakeDate.vb]: ok, thanks.I will add this again. I thought we might not need this option again.

**Comment by dannyparsons** [comment]
and the same here you now have changes to the plot region designers

**Comment by shadrackkibet** [comment]
I intended to make designer changes on dlg & sdg plot region here.I also added changes on dlgMakedate.

**Comment by dannyparsons** [comment]
ok yes that's fine, I was getting confused between the pull requests

**Pull request merged**